src/julia_atomics.h

```diff
@@ -59,7 +59,7 @@ enum jl_memory_order {
 /**
  * Cache line size
 */
-#if defined(_CPU_AARCH64_) && defined(_OS_DARWIN_) // Apple silicon has 128 cache lines
+#if (defined(_CPU_AARCH64_) && defined(_OS_DARWIN_)) || defined(_CPU_PPC64_)  // Apple silicon and PPC7+ have 128 cache lines
```

Comment:
```suggestion
#if (defined(_CPU_AARCH64_) && defined(_OS_DARWIN_)) || defined(_CPU_PPC64_)  // Apple silicon and PPC7+ have 128 byte cache lines
```
Not your fault, but while you're at it